.drone.yml

```diff
@@ -54,6 +54,16 @@ pipeline:
     commands:
       - litmus-wrapper
 
+  litmus-test-new-endpoint:
```

Comment:
Maybe name that litmus-new and rename the other litmus-old.

Reply:
And you can tell me if we can run the old and new api in parallel

Reply:
parallel against the same server: no
against the same server with a different user or the same user and a different folder: yes